langlib/langlib-test/src/test/resources/test-src/xmllib_test.bal

```diff
@@ -110,10 +110,13 @@ function testXmlIsComment() returns [boolean, boolean] {
         emptyConcatCall() is 'xml:Comment];
 }
 
-function testXmlIsText() returns [boolean, boolean] {
+function testXmlIsText() returns [boolean, boolean, boolean, boolean, boolean] {
```

Comment:
I would add a new test (and do the assertions in Ballerina itself too).

Reply:
Fixed with cfc81cfad46986467573c70a6c7a316ea085d2ae